ngraph/core/src/op/reduce_mean.cpp

```diff
@@ -68,6 +69,8 @@ bool op::v1::ReduceMean::evaluate(const HostTensorVector& outputs,
                                   const HostTensorVector& inputs) const
 {
     NGRAPH_OP_SCOPE(v1_ReduceMean_evaluate);
+    NGRAPH_CHECK(validate_host_tensor_vector(inputs, 2));
+    NGRAPH_CHECK(validate_host_tensor_vector(outputs, 1));
     return mean::evaluate_mean(inputs[0], outputs[0], get_reduction_axes(), get_keep_dims());
```

Comment:
Please update to reuse `get_normalized_axes_from_tensor` helper

Reply:
Done